tsl/test/expected/dist_compression.out

```diff
@@ -1176,7 +1222,7 @@ INSERT INTO test_defaults SELECT '2001-01-01', 1;
 SELECT compress_chunk(show_chunks) AS compressed_chunk FROM show_chunks('test_defaults') ORDER BY show_chunks::text LIMIT 1;
```

Comment:
```suggestion
SELECT CASE WHEN compress_chunk(show_chunks) IS NOT NULL THEN 'compress' END AS compressed_chunk FROM show_chunks('test_defaults') ORDER BY show_chunks::text LIMIT 1;
```
This would get rid of the chunk numbers from test file